app/assets/stylesheets/framework_and_overrides.css.scss

```diff
@@ -4,14 +4,26 @@
 $body-font-family:   'sonarnormal', sans-serif;
 $header-font-family: 'sonarnormal', sans-serif;
 
-$eurucampRed: #f6271b;
-$eurucampYellow: #fafb47;
+$eurucampRed: #E500FF;
```

Comment:
![screenshot from 2015-03-14 19 29 24](https://cloud.githubusercontent.com/assets/51264/6652947/7a88d5c8-ca80-11e4-8ad9-562b1ce84eea.png)
That's a pretty liberal interpretation of »red« :smiley_cat: